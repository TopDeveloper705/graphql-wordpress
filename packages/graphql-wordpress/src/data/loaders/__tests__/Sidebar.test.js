import getSidebarLoaders from 'data/loaders/Sidebar';

const Sidebar = getSidebarLoaders();

describe('Test Sidebar loader', () => {
  test('Load a Sidebar', async () => {
    const sidebar = await Sidebar.load('main-nav');
    expect(sidebar.getID()).toMatchSnapshot();
    expect(sidebar).toMatchSnapshot();
  });
});
