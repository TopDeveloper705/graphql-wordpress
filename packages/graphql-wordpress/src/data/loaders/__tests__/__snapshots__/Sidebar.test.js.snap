// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Sidebar loader Load a Sidebar 1`] = `"U2lkZWJhcjo="`;

exports[`Test Sidebar loader Load a Sidebar 2`] = `
Sidebar {
  "slug": "main-nav",
}
`;
